### PR TITLE
BUG003: Fix register student issue

### PR DESCRIPTION
**Pull request by ldvo**
Updated student registration to use new db models.

### CODE REVIEW

File: app/models/student.rb [hunk 1]
```diff
@@ -8,5 +8,6 @@ class Student < ApplicationRecord
   has_many :scholarship_oportunities, through: :scholarships
 
   accepts_nested_attributes_for :contact_information
+  accepts_nested_attributes_for :scholarships
```
Comment (ReneSG) [inline app/models/student.rb:11]: You can do this in the line above. `accepts_nested_attributes_for :contact_information, :scholarships`

File: app/views/devise/registrations/new_with_student.html.erb [hunk 2]
```diff
@@ -27,133 +27,135 @@
...
             <h5>
-              Domicilio:
+              Información de Estudios:
             </h5>
+            <%= s.fields_for :scholarship_attributes do |sch| %>
             <div class="row">
               <div class="col s3 input-field inline">
-                <%= c.label :Calle_de_Domicilio,:style=>'color:red;' %>
-                <%= c.text_field :street_address, :required => true  %>
+                <%= select_tag 'scholarship_oportunity_id', options_for_select(ScholarshipOportunity.all.collect {|p| [ p.name, p.id ] }.insert(0, "Convocatoria")),:style=>'color:red;', :required => true ,class: "browser-default"%>
```
Comment (ReneSG) [inline app/views/devise/registrations/new_with_student.html.erb:87]: Does this work? Would it allow the user to select Convocatoria as an option?

Comment (ldvo) [reply app/views/devise/registrations/new_with_student.html.erb:87]: Yes (that was already there, I just fixed the formatting).

**Comment by ldvo** [comment]
Fixed a lot of issues with the student creation using old fields from db (fields not being saved).

**Pull request merged**